src/views/ContributionAssistant.vue

```diff
@@ -209,84 +209,133 @@ export default {
       // Summary tab should be enabled when there are product prices to be added and the add prices process is either running or done
       const enableSummaryTab = this.productPriceForms.length && (this.addPricesLoading || this.allDone)
       return !enableSummaryTab
+    },
+    proofsIdsFromQueryParam() {
+      return JSON.parse(this.$router.currentRoute.value.query.proofs)
```

Comment:
could we use `proof_ids=id1,id2` ?
- rename proofs to proof_ids
- use comma-seperated values

Reply:
an alternative could also be `proof_id=id1&next_proof_id=id2`

Reply:
Sure, I spend a bit of time looking for a standardized way of passing arrays in url params. It's quite surprising that there isn't. I went with this one for ease of use using JSON, but I do not like it either 😅

Reply:
proof_ids=id1,id2 has my preference, we already use it for product lists in ProductOpener

Reply:
changed here: https://github.com/openfoodfacts/open-prices-frontend/pull/1166/commits/07711e7b2ea99e31dd543ef85d7c0bac1290d3dd